backend/danswer/db/chat.py

```diff
@@ -484,6 +531,27 @@ def get_doc_query_identifiers_from_model(
     return doc_query_identifiers
 
 
+def update_search_docs_table_with_relevance(
+    db_session: Session,
+    reference_db_search_docs: list[SearchDoc],
+    relevance_summary: LLMRelevanceSummaryResponse,
+) -> None:
+    for search_doc in reference_db_search_docs:
+        relevance_data = relevance_summary.relevance_summaries.get(
+            f"{search_doc.document_id}-{search_doc.chunk_ind}"
```

Comment:
better to use the chunk.unique_id here too, better to have this stuff standardized instead of magically constructed strings

Reply:
Sent a ping about this one!